File: mpp_proiect_java/src/main/java/triatlon/domain/Arbitru.java
package triatlon.domain;

import java.time.LocalDateTime;

public class Arbitru extends Entity<Long>{

    private String firstName;
    private String lastName;
    private String email;
    private String userName;
    private String password;


    public Arbitru(Long idArbitru,String firstName, String lastName,String email,String password,String userName){
        setId(idArbitru);
        this.firstName = firstName;
        this.lastName = lastName;
        this.email = email;
        this.userName = userName;
        this.password = password;

    }
    public String getFirstName() {
        return firstName;
    }

    public void setFirstName(String firstName) {
        this.firstName = firstName;
    }

    public String getLastName() {
        return lastName;
    }

    public void setLastName(String lastName) {
        this.lastName = lastName;
    }

    public String getEmail() {
        return email;
    }

    public void setEmail(String email) {
        this.email = email;
    }

    public String getUserName() {
        return userName;
    }

    public void setUserName(String userName) {
        this.userName = userName;
    }

    public String getPassword() {
        return password;
    }

    public void setPassword(String password) {
        this.password = password;
    }


}

File: mpp_proiect_CSharp/triatlon/triatlon/domain/Arbitru.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace triatlon.domain
{ 
    class Arbitru : Entity<long>
    {
        public long Id { get; set; }

        public string firstName { get; set; }

        public string lastName { get; set; }

        public string email { get; set; }

        public string username { get; set; }

        public string password { get; set; }
  
        public Arbitru(long id, string firstname, string lastname, string email1, string username1, string password1)
        {
            Id = id;
            firstName = firstname;
            lastName = lastname;
            email = email1;
            username = username1;
            password = password1;

        }
        public override string ToString()
        {
            return "Arbitru - nume: " + firstName + '|' +
                "prenume=" + lastName;
        }

    }
}
